nasa/cFS-GroundSystem
# Fix #29, fix invalid length in ground system

**Describe the contribution**
Fix issue 29, invalid cmd length errors. 

**Testing performed**
Steps taken to test the contribution:
1. Build cFS
2. Build GroundSystem
3. Run groundsystem and send command
4. Verify cmd length errors does not appear.

**Expected behavior changes**
No impact to behavior

**System(s) tested on:**
 - Hardware
 - OS: Ubuntu 18.04.03
 - CFE 6.6.0, GroundSystem 2.0.90

**Additional context**
None

**Contributor Info**
Anh Van, NASA Goddard

**Community contributors**
You must attach a signed CLA (required for acceptance) or reference one already submitted
